launch/tier4_map_launch/launch/map.launch.xml

```diff
@@ -15,6 +16,7 @@
     <include file="$(find-pkg-share map_loader)/launch/pointcloud_map_loader.launch.xml">
       <arg name="pointcloud_map_path" value="$(var pointcloud_map_path)"/>
       <arg name="pointcloud_map_loader_param_path" value="$(var pointcloud_map_loader_param_path)"/>
+      <arg name="lanelet2_map_loader_param_path" value="$(var lanelet2_map_loader_param_path)"/>
```

Comment:
"pointcloud_map_loader.launch.xml" really expect this argument "lanelet2_map_loader_param_path"?